docs/AUGUR_LOGIN.md

```diff
@@ -0,0 +1,58 @@
+# **8Knot+Augur User Sessions**
+
+## **Intro**
+---
+
+With this feature, users can:
+1. Create an account on an Augur database frontend page.
+2. Organize repos and orgs into named groups in Augur, specific to their user account.
+3. Log in to to Augur via 8Knot and see aggregate visualizations of their named groups.
+
+## **Setup 8Knot to enable Login and user groups**
+---
+1. Clone [github.com/oss-aspen/8Knot/dev]
+   - We use a docker compose multi-container application build strategy.
+   - Ports 8050 (*Dash*) and 6379 (*Redis*) are assumed to be available. If this isn't possible, modifications to the docker-compose.yml file may be necessary.
+   - Inter-pod name resolution is done with Docker network's DNS.
+
+2. Create an application in Augur front-end
```

Comment:
I would specific in here that this is an "admin'"/"developer" log in. I remember the whole process was confusing to me until I understood which part was "admin'"/"developer"  and which parts was a user log in

Reply:
Okay understood, will update